unicode-math-table.tex

```diff
@@ -822,7 +822,9 @@
 \UnicodeMathSymbol{"027C7}{\veedot                   }{\mathbin}{or with dot inside}%
 \UnicodeMathSymbol{"027C8}{\bsolhsub                 }{\mathrel}{reverse solidus preceding subset}%
 \UnicodeMathSymbol{"027C9}{\suphsol                  }{\mathrel}{superset preceding solidus}%
+\UnicodeMathSymbol{"027CB}{\diagup                   }{\mathord}{mathematical rising diagnal}%
```

Comment:
Unicode  http://www.unicode.org/reports/tr25/ mathclass15.tx has these as class R and MathML Operator dictionary gives them spacing, so if they are added, they should probably be `\mathrel` not `\mathord`

That said, most 8bit math font packges make the command `\diagup` `\mathord`

```
amsfonts/amssymb.sty:\DeclareMathSymbol{\diagup}       {\mathord}{AMSb}{"1E}
arev/ams-mdbch.sty:    \DeclareMathSymbol{\diagup}       {\mathord}{mathdesignA}{"1E}
boisik/boisik.sty:\DeclareMathSymbol{\diagup}               {\mathord}{bsksymbols}{"DB}
breqn/msabm.sym:\DeclareFlexSymbol{\diagup}          {Ord}{MSB}{1E}
```

with MSymbol the exception

```
mnsymbol/MnSymbol.sty:  \Decl@Mn@Alias\diagup              {MnSyA}{\mathrel}
mnsymbol/MnSymbol.sty:  \Decl@Mn@Alias\ndiagup             {MnSyB}{\mathrel}
```

several OTF seem to have this in the private use E0A0 slot

```
concmath-otf/concmath-otf.sty:     \Umathchardef\diagup =         0 0 "E0A0 \relax
erewhon-math/fourier-otf.sty:     \Umathchardef\diagup =         0 0 "E0A0 \relax
xcharter-math/xcharter-otf.sty:     \Umathchardef\diagup =         0 0 "E0A0 \relax
```

Font coverage is a bit variable


```
\documentclass{article}
\usepackage{fontspec}
\newfontfamily{\lmm}{Latin Modern Math}
\newfontfamily{\stm}{STIX Two Math}
\newfontfamily{\tgp}{TeX Gyre Pagella Math}
\newfontfamily{\tgt}{TeX Gyre Termes Math}
\newfontfamily{\ncm}{NewCMMath-Regular.otf}
\newfontfamily{\ghm}{GFSNeohellenicMath.otf}
\newfontfamily{\cambriam}{Cambria Math}
\newfontfamily{\xcm}{XCharter-Math.otf}
\newfontfamily{\concm}{Concrete-Math.otf}

\begin{document}
\makeatletter

{\lmm [\char"2751][\char"27CB][\char"E0A0] \f@family\par}
{\stm [\char"2751][\char"27CB][\char"E0A0] \f@family\par}
{\tgp [\char"2751][\char"27CB][\char"E0A0] \f@family\par}
{\tgt [\char"2751][\char"27CB][\char"E0A0] \f@family\par}
{\ncm [\char"2751][\char"27CB][\char"E0A0] \f@family\par}
{\ghm [\char"2751][\char"27CB][\char"E0A0] \f@family\par}
{\cambriam [\char"2751][\char"27CB][\char"E0A0] \f@family\par}
{\xcm [\char"2751][\char"27CB][\char"E0A0] \f@family\par}
{\concm [\char"2751][\char"27CB][\char"E0A0] \f@family\par}


\end{document}
```

![image](https://user-images.githubusercontent.com/1268738/199455273-428bb299-de1c-40c8-bfcd-9b5899ca2aad.png)

But I suppose adding here might encourage fonts to converge, but mathord or mathrel? amssymb compatbilty means I think the ame `\diagup` should be mathord, but I wonder if a mathrel version should be supplied as well...